spec/models/web_search_spec.rb

```diff
@@ -334,44 +338,77 @@
 
       context 'when we want X Bing/Google results from page Y and there are 0 <= n < X of them' do
         let(:search) do
-          described_class.new(query: 'odie backfill page 2', affiliate: affiliate, page: 2)
+          described_class.new(query: 'odie backfill', affiliate: affiliate, page: page)
         end
+        let(:page) { 1 }
 
         before do
           ElasticIndexedDocument.recreate_index
 
           bing_api_url = "#{BingV6WebSearch::API_HOST}#{BingV6WebSearch::API_ENDPOINT}"
-          page2_6results = Rails.root.join('spec/fixtures/json/bing_v6/web_search/page2_6results.json').read
-          stub_request(:get, /#{bing_api_url}.*odie backfill page 2/).
-            to_return(status: 200, body: page2_6results)
+          page1_6results = Rails.root.join('spec/fixtures/json/bing_v6/web_search/page1_6results.json').read
```

Comment:
I noticed this is using Bing V6. Do we need to update these specs to use V7?

Reply:
I was attempting to avoid having to re-stub a BingV7-style response, but you're right... I've updated to V7, deleted the old stubbed json response, and added a new v7 response.